ppx_src/bin/Utils.re

```diff
@@ -9,22 +8,34 @@ let decoderFuncSuffix = "_decode";
 let encoderVarPrefix = "encoder_";
 let decoderVarPrefix = "decoder_";
 
+let loc = default_loc^;
+
 let fail = (loc, message) =>
-    Location.error(~loc, message)
-    |> (v) => Location.Error(v)
-    |> raise;
+    Location.raise_errorf(~loc, "%s", message);
+    /* |> (v) => Location.Error(v) */
+    /* |> raise; */
```

Comment:
Is `Location.raise_errorf` equivalent to the previous code? Is it necessary to keep this commented code around?

Reply:
it is. I don't know why I kept it!